custom_components/tuya_local/devices/neopower_heat_pump_water_heater.yaml

```diff
@@ -0,0 +1,96 @@
+name: Neopower All in One Heat Pump
```

Comment:
Top level name should be without branding. If you can find the product_id for this device (from the same platform.tuya.com API calls that return the local key, or from a local discovery scan by tinytuya etc), then a products section can be added with id, manufacturer and model details.